### PR TITLE
Added support for tailoring file UI

### PR DESCRIPTION
**Pull request by ntkathole**
Test Results 
```
pytest tests/foreman/ui/test_oscap_tailoringfiles.py::TailoringFilesTestCase::test_positive_create
=========================== test session starts ==================================
collected 14 items

tests/foreman/ui/test_oscap_tailoringfiles.py .

============================== 1 passed ======================================
```

### CODE REVIEW

File: tests/foreman/ui/test_oscap_tailoringfiles.py [hunk 1]
```diff
@@ -43,12 +58,24 @@ def test_positive_create(self):
             2. Upload a valid tailoring file
             3. Give it a valid name
 
-        :CaseAutomation: notautomated
+        :CaseAutomation: automated
```
Comment (oshtaier) [inline tests/foreman/ui/test_oscap_tailoringfiles.py]: we do not use such tag, we just remove it at all =)

File: robottelo.properties.sample [hunk 2]
```diff
@@ -218,6 +218,7 @@ ssh_key=
 # For OSCAP Testing
 # [oscap]
 # content_path=~/ssg-rhel6-ds.xml
+# tailoring_path=~/ssg-firefox-ds-tailoring.xml
```
Comment (san7ket) [inline robottelo.properties.sample]: You need to raise a PR in gitlab for this change.

**Comment by san7ket** [comment]
Bump @ntkathole tailroing_path is added to our properties file in gitlab , please re-run the test with the same config file

**Comment by ntkathole** [comment]
@san7ket , gitlab PR 83 merged.

**Comment by san7ket** [comment]
@ntkathole re-run test with the new properties

**Comment by ntkathole** [comment]
@oshtaier  and @san7ket  Results after re-run : 

```
 pytest tests/foreman/ui/test_oscap_tailoringfiles.py::TailoringFilesTestCase::test_positive_create
============================================= test session starts =============================================

collected 14 items
2017-08-16 18:44:37 - conftest - DEBUG - Deselect of WONTFIX BZs is disabled in settings


tests/foreman/ui/test_oscap_tailoringfiles.py .

====================================== 1 passed ====================================
```

**Comment by san7ket** [comment]
Merging this, two acks

**Pull request merged**